package.json

```diff
@@ -7,7 +7,7 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "jest",
-    "openapi": "openapi-generator-cli generate -i ./openapi.json -g typescript-angular -o src/app/api/generated"
+    "openapi": "openapi-generator-cli generate -i ./openapi.json -g typescript-angular -o src/app/api/generated --skip-validate-spec"
```

Comment:
Als ik --skip-validate-spec weg laat dan krijg ik de volgende error:
```
-attribute components.securitySchemes.Auth0.scopes is missing
```
Ik snap dus waarom dit nu uit staat, maar is dit iets dat we nog willen fixen? Zo ja, is daar een ticket voor? Dat ticket, danwel de uitleg waarom we validatie skippen, kunnen we documenteren in de README.

Reply:
Ik kreeg juist een error zonder die `skip-validate-spec` haha. Ik ben niet. zo bekend met deze generator. Misschien iets waar Marijn eens in kan duiken en kan documenteren?